### DISCUSSION TITLE
Disable "(INTEGER_DIVISION): Integer division, decimal part will be discarded)"

### DISCUSSION THREAD

**Discussion by gerroon** [godotengine/godot-vscode-plugin#619] [Q&A] [answered]
Hi

Is there a way to disable some of these warnings the wiggly lines in VsCode? I am ok with some of these, they do not need to catch my attention at all times.

thanks

**Comment by DaelonSuzuka** [comment]
Just turn it off?

https://docs.godotengine.org/en/stable/classes/class_projectsettings.html#class-projectsettings-property-debug-gdscript-warnings-integer-division

**Reply by gerroon** [reply]
Thanks for the reply. I did not know that so I set this in project.godot (at the bottom) I still get that wiggly warning in VsCode.
```
[editor]
version_control/plugin_name="GitPlugin"
version_control/autoload_on_startup=true
debug/gdscript/warnings/integer_division=0
```

![image](https://github.com/godotengine/godot-vscode-plugin/assets/8519469/c89c064f-4745-43e0-9f59-9f51b3f37dc0)

**Reply by DaelonSuzuka** [reply, marked as answer]
Why would you do that? That's the wrong section and the wrong format for the setting. 

Just open Project Settings in the Godot Editor and change the setting normally.

After you do that, restart the Godot Editor and VSCode to make sure everything resets properly.

**Reply by gerroon** [reply]
"Why would you do that? That's the wrong section and the wrong format for the setting." Well because I thought that the documentation said to do. 

Thanks for the help, I appreciate it. I will give it a try.

Also thanks for the extension, this really makes it a more fluid workfow.

**Reply by DaelonSuzuka** [reply]
Sorry for the confusion, then. I assumed any Godot user would be familiar with Project Settings.

> Also thanks for the extension, this really makes it a more fluid workfow.

Hey, thank you! I'm glad you're finding it useful.

**Reply by Calinou** [reply]
> Well because I thought that the documentation said to do.

The path displayed in the ProjectSettings class reference *is* valid, but you need to put it in the right *section* (like in INI files). The section in the file is denoted by the text between brackets located above the line, so you actually defined `editor/debug/gdscript/warnings/integer_division` to `0`.

You could make the change to `project.godot` while the editor is closed, but you'd have to add the line under the `[debug]` section or create a `[debug]` section if it doesn't exist already.

```ini
[debug]

debug/gdscript/warnings/integer_division=0
```
